docker/Dockerfile.bionic

```diff
@@ -0,0 +1,41 @@
+# DockerFile for a Fludity development container
+
+# Use a Bionic base image
+FROM ubuntu:bionic
+
+# This DockerFile is looked after by
+MAINTAINER Tim Greaves
+
+# Installs shouldn't expect input
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Update the system
+RUN apt-get update
+RUN apt-get -y dist-upgrade
+
+# Install gpg
+RUN apt-get -y install gnupg dirmngr
+
+# Add the Fluidity repository
+RUN echo "deb http://ppa.launchpad.net/fluidity-core/ppa/ubuntu bionic main" > /etc/apt/sources.list.d/fluidity-core-ppa-bionic.list
+
+# Import the Launchpad PPA public key
+RUN gpg --keyserver keyserver.ubuntu.com --recv 0D45605A33BAC3BE
+RUN gpg --export --armor 33BAC3BE | apt-key add -
+RUN apt-get update
+
+# Install Fluidity development environment
+RUN echo "Europe/London" > /etc/timezone
```

Comment:
This is an annoying issue where, in a minimal system, when you install tzdata it needs either to interact to determine the timezone or to have a pre-defined timezone on the system. It's a little while since I looked at this but I think if you haven't pre-defined the timezone and you've specified a non-interactive install the package install just stops with an unresolveable state.